### PR TITLE
Add verbose option to run_apex_tests task

### PR DESCRIPTION
**Pull request by bethbrains**
# Critical Changes

# Changes
- `run_apex_tests` now defaults to only output class and method names of failed tests.
- Setting new boolean option `verbose` to True will result in output of all class and method names and their respective results.

# Issues Closed

### CODE REVIEW

**Comment by bethbrains** [comment]
Admittedly super tiny annoyance, but deeply felt as we started using `run_apex_tests` more with the coverage enforcement feature.

File: cumulusci/tasks/apex/tests/test_apex_tasks.py [hunk 1]
```diff
@@ -468,6 +470,42 @@ def test_run_task__processing(self):
         log = self._task_log_handler.messages
         assert "Completed: 0  Processing: 1 (TestClass_TEST)  Queued: 0" in log["info"]
 
+    @responses.activate
+    def test_run_task__not_verbose(self):
+        # self._task_log_handler.reset()
```
Comment (davisagli) [inline cumulusci/tasks/apex/tests/test_apex_tasks.py:475]:
```suggestion
```

File: cumulusci/tasks/apex/tests/test_apex_tasks.py [hunk 2]
```diff
@@ -468,6 +470,42 @@ def test_run_task__processing(self):
...
+        self._mock_apex_class_query()
+        self._mock_run_tests()
+        self._mock_tests_processing()
+        self._mock_get_failed_test_classes()  # this returns all passes
+        self._mock_tests_complete()
+        self._mock_get_test_results()
+        task = RunApexTests(self.project_config, self.task_config, self.org_config)
+        task()
+        log = self._task_log_handler.messages
+        print(log["info"])
```
Comment (davisagli) [inline cumulusci/tasks/apex/tests/test_apex_tasks.py:485]:
```suggestion
```

**Comment by bethbrains** [comment]
@davisagli all requested changes (and oops fixes) made. thank you! no rush; for your post-falcon perusal.

**Pull request merged**